dali/pipeline/operators/detection/box_encoder_test.cc

```diff
@@ -1062,8 +1065,14 @@ class BoxEncoderTest : public GenericBBoxesTest<ImgType> {
         auto expected = offset ? coco_offsets[sample][i] : match.second.first;
 
         ASSERT_EQ(labels_data[match.first], match.second.second);
-        ASSERT_FLOAT_EQ(actual.x, expected.x);
-        ASSERT_FLOAT_EQ(actual.y, expected.y);
+
+        if (!offset) {
+          ASSERT_FLOAT_EQ(actual.x, expected.x);
+          ASSERT_FLOAT_EQ(actual.y, expected.y);
+        } else {
+          ASSERT_NEAR(actual.x, expected.x, 0.00001);
+          ASSERT_NEAR(actual.y, expected.y, 0.00001);
```

Comment:
GPU version has slightly different results then the CPU one due to numerical errors. Unlike before, here this is not impacting the anchor matched with the box, only the values are different. I tried to get the exact same results with `volatile` and stuff, but couldn't do it. The differences are very small though and should not impact the training.

Reply:
>  should not impact the training.
Only one way to know :stuck_out_tongue:

Reply:
My thoughts exactly :)